ekip/everykid/templates/educator_form.html

```diff
@@ -9,7 +9,7 @@
     {% for field in form %}
       {% if field.html_name == 'organization_name' %}
     </ul>
-    <span>Organizational info:</span>
```

Comment:
I think this should say Organizational Information (spell the word out). 
As for the student bit, maybe we need to move that field elsewhere.